hack/generate/samples/ansible/memcached.go

```diff
@@ -22,17 +22,17 @@ import (
 	log "github.com/sirupsen/logrus"
 	kbtestutils "sigs.k8s.io/kubebuilder/test/e2e/utils"
 
-	"github.com/operator-framework/operator-sdk/hack/generate/samples/internal/pkg"
+	"github.com/operator-framework/operator-sdk/internal/samples"
```

Comment:
IMO we should avoid separate tooling-only code from code used in the SDK binary. Is there a reason this package move was necessary?

Reply:
We need to use the `"github.com/operator-framework/operator-sdk/hack/generate/samples/internal/pkg"` in the e2e tests. Then, it is not possible if it is in internal dir in the hack. So, the change here was only moved to "github.com/operator-framework/operator-sdk/internal/samples" instead of letting it exported. Just following your approach in another pr where was created the `operator-sdk/internal/testutils/`